lib/run.js

```diff
@@ -14,43 +14,13 @@ var parse = require("shell-quote").parse;
  * @param {Object} options
  *   - `binary` path to Firefox binary to use
  *   - `profile` path to profile or profile name to use
+ *   - `binary-args-first` put the binary arguments first in the list of
+ *     command arguments
```

Comment:
@willdurand if I'm reading this correctly (also combined with https://github.com/mozilla/web-ext/pull/2477 which depends on this change): 
- for Firefox CLI arguments having the binary args appended at the end works just fine, and the order doesn't matter that much 
- if we are using node-fx-runner to run a wrapper of the actual Firefox binary, like we have to do in https://github.com/mozilla/web-ext/pull/2477 to run Firefox through `flatpak run ...`, then we need that the binary args are coming first of any other Firefox CLI option that node-fx-runner adds based on the rest of the options we are passing it

it may be worth leaving for ourselves a note here about why/when we will be using binary-args-first.

Reply:
Yeah, I kept the old behavior just in case it matters but I am not sure it does... And I wasn't around when that was introduced obviously :D